com.unity.render-pipelines.core/Documentation~/Look-Dev.md

```diff
@@ -14,7 +14,7 @@ You can load two different Assets into Look Dev at the same time and compare the
 
 ## Using Look Dev
 
-To open Look Dev in the Unity Editor, select **Window > Render Pipeline > Look Dev**. The first time you use Look Dev, you must either create a new [Environment Library](Look-Dev-Environment-Library.html) or load one. For information on how to create an Environment Library, see the [Environment Library documentation](Look-Dev-Environment-Library.html).
+To open Look Dev in the Unity Editor, select **Window > Analysis > Look Dev**. The first time you use Look Dev, you must either create a new [Environment Library](Look-Dev-Environment-Library.html) or load one. For information on how to create an Environment Library, see the [Environment Library documentation](Look-Dev-Environment-Library.html).
```

Comment:
Was this forgotten to be updated to "Rendering" instead of "Analysis"?

Reply:
Changed, nice catch :)